src/main/java/baron/messages/Messages.java

```diff
@@ -8,15 +8,20 @@
     public static final String MESSAGE_DELETE_SUCCESS = "Noted. I've removed this task:\n ";
     public static final String MESSAGE_ADD_TASK_SUCCESS = "Got it. I've added this task:\n  ";
     public static final String MESSAGE_LIST_TASK = "Here are the tasks in your list:\n";
-    public static final String MESSAGE_NO_TASK = "You have no tasks.\n";
-    public static final String MESSAGE_FILE_NOT_FOUND = "Data file is not found.\n";
-    public static final String MESSAGE_INVALID_FILE_FORMAT = "Data file format is invalid.\n";
-    public static final String MESSAGE_FILE_CREATION_FAIL = "Data file cannot be created.\n";
-    public static final String MESSAGE_FILE_WRITE_FAIL = "Data file cannot be written to.\n";
+    public static final String MESSAGE_NO_TASK = "You have no tasks.";
+    public static final String MESSAGE_FILE_NOT_FOUND = "Data file is not found.";
+    public static final String MESSAGE_INVALID_FILE_FORMAT = "Data file format is invalid.";
+    public static final String MESSAGE_FILE_CREATION_FAIL = "Data file cannot be created.";
+    public static final String MESSAGE_FILE_WRITE_FAIL = "Data file cannot be written to.";
     public static final String MESSAGE_DATE_TIME_FORMAT_INVALID = "The date/time should be in the following format: d/M/yyyy HH:mm";
```

Comment:
I didn't know you could name all the default messages as an attribute, I'm impressed!

Reply:
Hi De Shao, my reason for this design is actually just for me to easily change the outputs, instead of finding different Java files to fix the outputs. For example, if I want to change to a more kids-friendly tone. Glad that you are impressed!